checks.d/agent_metrics.py

```diff
@@ -113,6 +114,11 @@ def get_metric_context(self):
         return self._collector_payload, self._metric_context
 
     def check(self, instance):
+
+        if Platform.is_linux():
+            procfs_path = self.agentConfig.get('procfs_path', '/proc').rstrip('/')
+            psutil.PROCFS_PATH = procfs_path
```

Comment:
one other thing: in this check we can't assume that `psutil` has been imported successfully, so we have to check that `psutil is not None` before setting `psutil. PROCFS_PATH`.

Especially important in this check since it's run by default with the agent.